Content.IntegrationTests/Tests/Weapons/WeaponSniperTests.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Content.IntegrationTests.Tests.Interaction;
+using Content.Shared.Damage;
+using Content.Shared.Weapons.Ranged.Components;
+using Content.Shared.Weapons.Ranged.Events;
+using Content.Shared.Wieldable.Components;
+using Robust.Shared.Prototypes;
+
+namespace Content.IntegrationTests.Tests.Weapons;
+
+public sealed class WeaponSniperTests : InteractionTest
```

Comment:
```suggestion
public sealed class WeaponTests : InteractionTest
```
We will put other weapon tests in here later. Same for the file name.